docs/source/package-spec.rst

```diff
@@ -518,13 +505,69 @@ for the following common resources.
   - ``documentation``: Package Documentation
   - ``repository``: Location of the project source code.
 
-
   :Key: ``links``
 
   :Type: Object (String: String)
 
 ----
 
+.. _Sources Object:
+
+The *Sources* Object
+~~~~~~~~~~~~~~~~~~~~
+
+A *Sources* object is defined to have the following fields.
+
+  :Key: A global identifier for the source file. (string)
```

Comment:
I think this suggests that all keys should be unique but doesn't state it explicitly.  Maybe it is worth having a global rule that all objects **must** have a unique set of keys unless otherwise specified.

Reply:
Definitely - we have a rule like that already defined in the document formatting section: `Duplicate keys in the same object are invalid.`.